### PR TITLE
Fix disabled

### PR DESCRIPTION
**Pull request by blasten**
Issue: https://github.com/PolymerElements/paper-slider/issues/57.
Dependency: https://github.com/PolymerElements/paper-progress/pull/20

### CODE REVIEW

**Comment by frankiefu** [comment]
LGTM

File: paper-slider.html [hunk 1]
```diff
@@ -441,23 +441,15 @@
...
-      classes.expand = this.expand;
-      classes.dragging = this.dragging;
-      classes.transiting = this.transiting;
-      classes.editable = this.editable;
-
-      return this._mergeClasses(classes);
-    },
-
-    _getProgressClass: function() {
       return this._mergeClasses({
```
Comment (samccone) [inline paper-slider.html:444]: any thoughts on the fact that we are using a private API method here¿

Comment (blasten) [inline paper-slider.html:444]: `_mergeClasses` is defined in paper-slider. I believe there's now a helper function in Polymer. I will check.

Comment (samccone) [inline paper-slider.html:444]: :+1: sgtm :)

**Pull request merged**